### PR TITLE
Build from salt-windows-nsis

### PR DESCRIPTION
**Pull request by twangboy**
The build scripts for the Nullsoft installer are being pulled out of the Salt project into their own repo. The MSI builder will need to look there for the built Nullsoft installer as well as `buildenv` and `configs`.

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -48,7 +48,10 @@ $WEBCACHE_DIR = "$pwd\_cache.dir"
 ## Verify or install WiX toolset from https://wixtoolset.org/releases
 ##   Wix 3.11.2  released Sep 19, 2019
 
-if (ProductcodeExists "{03368010-193D-4AE2-B275-DD2EB32CD427}") {
+# 64bit: {03368010-193D-4AE2-B275-DD2EB32CD427}
```
Comment (marbx) [inline build.ps1:51]: 64bit WiX can create both 64 and 32bit msi.
It seems you want to use both 64bit and 32bit WiX, which is fine, but I never tried.

File: build.ps1 [hunk 2]
```diff
@@ -155,20 +163,20 @@ Write-Host -ForegroundColor Green "Internal version  $internalversion"
 
 #### Detecting target platform from NSIS exe
 $targetplatform = 0
-if (Test-Path ..\salt\pkg\windows\installer\Salt-Minion*AMD64*.exe) {$targetplatform="64"}
-if (Test-Path ..\salt\pkg\windows\installer\Salt-Minion*x86*.exe)   {$targetplatform="32"}
+if (Test-Path ..\salt-windows-nsis\build\Salt-Minion*AMD64*.exe) {$targetplatform="64"}
```
Comment (marbx) [inline build.ps1:166]: The most obvious change needed

File: build.ps1 [hunk 3]
```diff
@@ -182,8 +190,8 @@ $PRODUCT        = "Salt Minion"
 $PRODUCTFILE    = "Salt-Minion-$displayversion"
 $PRODUCTDIR     = "Salt"
 $VERSION        = $internalversion
```
Comment (marbx) [inline build.ps1:192]: The second and last reference to the Nullsoft repo.

**Comment by marbx** [comment]
Looks good to me.
I have no Windows PC on which I could run a smoke test with the msi, but I assume you did.

Go for it!

**Comment by marbx** [comment]
All my code comments are remarks.

I don't see an "approve" button, please consider the PR approved.

**Comment by marbx** [comment]
Could you please adapt one more file: `test-copy_mock_files_to_salt_repo.cmd`

This allows a developer to run the config tests outside of Jenkins.

**Comment by twangboy** [comment]
@marbx Updated `test-copy_mock_files_to_salt_repo.cmd` with the new locations

**Pull request merged**